playbooks/k8s-cluster/nvidia-gpu-operator.yml

```diff
@@ -3,7 +3,7 @@
 - include: ../bootstrap/bootstrap-openshift.yml
 
 # GPU operator
-- hosts: kube-master[0]
```

Comment:
The expectation is that Helm commands are run from the provisioning node. No need to install Helm and run it on the management systems.